src/Promotions/WelcomeBanner/WelcomeBanner.php

```diff
@@ -13,7 +13,11 @@ public static function isShowing(): bool
 
         $option = get_option('givewp_welcome_banner_dismiss');
 
-        return $pagenow === 'plugins.php' && !$option;
+        $newStartDate = strtotime('2024-02-26 00:00');
+
+        $showBanner = ! $option || ($option < $newStartDate);
```

Comment:
Wait, so we're going to show the banner all over again for folks that have previously seen it?

Reply:
I spoke with Angela. We are looking to reintroduce the banner each time we have a big, new feature.

Reference: https://lw.slack.com/archives/C04SLRDD9CK/p1708978256130769